docs/containers/quickstart-python.md

```diff
@@ -0,0 +1,142 @@
+
+After verifying your app runs properly, you can now Dockerize your application.
+
+## Add Docker files to the project
+
+1. Open the project folder in VS Code.
+2. Open the Command Palette (`kb(workbench.action.showCommands)`) and use **Docker: Add Docker Files to Workspace...** command:
+
+    ![Add Dockerfile to a Python project](images/quickstarts/python-add-python.png)
+
+3. When the prompt appears, select **Python: Django**, **Python: Flask**, or **Python: General** as the app type. For this tutorial, we will select **Python: Django**.
+4. Select either **Yes** or **No** when prompted to include [Docker Compose](https://docs.docker.com/compose/) files.
+
+5. Enter the relative path to the app’s entry point. This excludes the workspace folder you start from. According to [official Django documentation](https://docs.djangoproject.com/en/3.0/intro/tutorial01/#creating-a-project), this path is commonly `manage.py` (root folder) or `subfolder_name/manage.py`. According to [official Flask documentation](https://flask.palletsprojects.com/en/1.1.x/api/), this is the path to where you create your Flask instance.
+
+>**Tip**: You may also enter the path to a folder name as long as this folder includes a `__main__.py` file.
+
+6. If **Python: Django** or **Python: Flask** was selected, specify app port for local development. Django defaults to port 8000 while Flask defaults to port 5000, however, any unused port will work.
+
+7. With all this information, the Docker extension creates the following files:
+
+    - A `Dockerfile`. To learn more about Intellisense in this file, refer to the [overview](/docs/containers/overview.md).
+
+    - A `.dockerignore` file to reduce the image size by excluding files and folders that aren't needed such as `.git`, `.vscode`, and `__pycache__`.
+
+    - If Docker Compose was selected, a `docker-compose.yml` file.
+
+    - If one does not already exist, a `requirements.txt` file for capturing all app dependencies.
+    > **Tip**: If the virtual environment/host machine is meant to be identical to the container, ensure app dependencies are ported over by running `pip freeze > requirements.txt` in the terminal to overwrite this file.
+
+### File modifications for Django/Flask apps
+
+To give Python Web Developers a great starting point, we chose to use [Gunicorn](https://gunicorn.org/#docs) as the default web server. Since it is referenced in the default Dockerfile, it is included as a dependency in the `requirements.txt` file.
+
+> **Note**: To use Gunicorn as your web server, it must be included in the `requirements.txt` file as an app dependency. It does not need to be installed in your virtual environment/host machine.
+
+#### Django Apps
+
+To use Gunicorn, it must bind to an application callable as an entry point. This callable is declared in the `wsgi.py` file of a Django application. To accomplish this binding, the final line in the Dockerfile says:
```

Comment:
Application callable - will this term be known to all users?  Could it be explained?  "it must bind to an application callable as an entry point'.  It seems like callable is being used as a noun, but I think of it as an adjective.

Reply:
It's a fairly used term in WSGI, but I changed it to "... must bind to an application callable (what the application server uses to communicate with your code) as an entry point." for better clarity. Thanks